### PR TITLE
Update dcp-updates.md

### PR DESCRIPTION
**Pull request by ekiernan**
The links on this PR need to be confirmed once DCP 21 is live.

### CODE REVIEW

**Comment by ekiernan** [comment]
@kayleemathews @NoopDog Please review and we'll need to verify the links when the release is live.

File: content/community-update/community-update/dcp-updates.md [hunk 1]
```diff
@@ -6,6 +6,23 @@ title: DCP Updates
 
 # DCP Platform Updates
 
+#### October 19, 2022
+
+### Release Highlights
+* New COVID-19 data is available in the project ["COVID-19 tissue atlases reveal SARS-CoV-2 pathology and cellular targets."](https://data.humancellatlas.org/explore/projects/61515820-5bb8-45d0-8d12-f0850222ecf0)
+
+* The new project ["Genome-Wide DNA Hypermethylation in the Wound-Edge of Chronic Wound Patients Opposes Closure by Impairing Epithelial to Mesenchymal Transition"](https://data.humancellatlas.org/explore/projects/34ec845b-cd7a-4c43-99e4-d2932d5d85bb) combines spatial transcriptomic data with methylome data.
```
Comment (kayleemathews) [inline content/community-update/community-update/dcp-updates.md]: It looks like we have been inconsistent about using quotes around project names in the Release Highlights. It's not a big deal but we might want to revisit this for consistency.

**Review by kayleemathews — APPROVED** [review]
One small comment but LGTM once we check the links!

File: content/community-update/community-update/dcp-updates.md (hunk 1 above)
Comment (ekiernan) [reply content/community-update/community-update/dcp-updates.md]: Good point and thanks! Removed quotes for now around project titles.

**Review by NoopDog — APPROVED** [review]

**Pull request merged**